### DISCUSSION TITLE
Timeseries development

### DISCUSSION THREAD

**Discussion by gavinny** [NASA-GISS/freshwater-forcing-workshop#6] [Timeseries development]
The is for discussions related to the merging of data sources to create the input files and assessments of uncertainties.

**Comment by durack1** [comment]
@gavinny looking forward to this discussion, much to learn and calibrate!

**Comment by AndyShepCPOM** [comment]
**IMBIE**

_Spatial Resolution_
The IMBIE team will be increasing the spatial resolution of our reconciled ice sheet mass balance assessment to drainage basin scale over the coming months and we are very happy to contribute this dataset to the project. 

_Temporal resolution_
Our dataset will run from ~1980 to present day and will be annual in temporal resolution. If finer is required/desired please let us know

_Floating ice_
We would be interested to consider extending our assessment to include floating ice (ice shelves & ice tongues, not sea ice). It seems that there are a variety of datasets currently in use (Depoorter, Rignot, Adusimilli, Slater, Greene, etc) and so (similar to land ice mass balance) a reconciled estimate could be of value to the user community.

**Reply by torgemartin** [reply]
by Torge Martin: To assess and reconcile the different data sets of floating ice would be very useful to the modeling community, would be great to have an "optimal", easily usable go-to data set. To create this, we also need to clarify terminology, e.g. ice sheet surface liquid runoff, grounding line flux (= iceberg calving + ice shelf melt), mass loss as freshwater flux (liquid/solid) vs. sea-level equivalent, etc. There is of course a time lag between grounding line flux and calving/melting, which is currently often neglected, I think, when such estimates are used in ocean/climate models. 
What exactly can be derived from observations and how should this be transferred into a quantity that is useful to  ocean models. The latter are of varying complexity, for instance, groups may want to distinguish between solid discharge (icebergs) and liquid ice shelf melt influx.

**Reply by AndyShepCPOM** [reply]
Great; I will contact the authors of the various floating ice datasets I am aware of and try to get something off the ground. Within IMBIE we already have a surface mass balance group and so we can deal with the partitioning as well.

**Reply by mankoff** [reply]
:warning: Mass balance (the IMBIE product) and freshwater (the focus here) are totally different. It is possible to have positive mass balance and still have large freshwater contribution to the surrounding seas.

As for temporal resolution, seasonal or monthly would be nice - but again, perhaps irrelevant because of different products.

I am curious if the IMBIE team sees a way to connect their product to freshwater. You could have all contributors provide constituent terms, not just mass. This is only possible with the IO method, not gravimetric or volumetric estimates of mass change.

**Reply by torgemartin** [reply]
Thanks for clarifying @mankoff. These discrepancies (beyond simple terminology) need to be addressed so the data sets are actually useful and will be used correctly.

**Reply by AndyShepCPOM** [reply]
Even apples and oranges are not _totally_ different - both fruits and both spherical to first order ;) We can compute freshwater flux from the data contributed to IMBIE. However, the discussions in our group were today centred on anomalous freshwater flux, not the mean, so it would be helpful to have a steer from the modelling community as to what is desired. I guess both?

**Reply by torgemartin** [reply]
Sure, if we can have both, we take it :-) In the discussion we had earlier (Oc/Eu1) we mentioned to accept the ice sheets (Greenland and Antarctica) to be in balance prior to, say, 1970. So the anomalous freshwater flux we talk about would be any mass imbalance that occurred since then. However, as Ken pointed out above, one could have an overall mass gain and still anomalously large freshwater discharge into the ocean in some regions, e.g. Amundsen-Bellinghausen Seas. How regional can a mass imbalance be derived? Is it possible to have the mass imbalance per ice shelf (e.g. grounding line flux vs ice shelf thinning and area change)?

**Reply by AndyShepCPOM** [reply]
Grounded ice mass balance will be at drainage basin scale in the first instance which should I think be good enough for the major ice shelves/seas (Ross, Weddell, Amundsen, Bellingshausen etc). For floating ice mass balance I guess we will need to discuss with the contributors what might be possible. My guess is it can be done at a similar scale.

**Reply by AndyShepCPOM** [reply]
http://imbie.org/imbie-3/drainage-basins/

**Reply by mankoff** [reply]
> Thanks for clarifying @mankoff. These discrepancies (beyond simple terminology) need to be addressed so the data sets are actually useful and will be used correctly.

@torgemartin  I agree, the different groups and communities need a common language.  I tried to address this in my presentation. 

I've been on an ice breaker since late December mostly near Thwaites and have limited internet, so I can't send you a link to my presentation, but Gavin uploaded it to Google Drive in the Observation sub-folder. We're due back in NZ on the 15th so next week I should be able to watch all the videos and comment more.

**Reply by mankoff** [reply]
> We can compute freshwater flux from the data contributed to IMBIE

@AndyShepCPOM  - how? Can you do this from non-IO method?

> it would be helpful to have a steer from the modelling community as to what is desired. I guess both?

I would want total first, mean + anomaly second, and anomaly-only third.

**Reply by AndyShepCPOM** [reply]
We have several input output (IO) datasets submitted to IMBIE and so it can be computed from those, but we also have surface mass balance (SMB) and so it can be computed from altimetry or gravimetry total mass balance using that (analagous to computing mass balance from IO method, which is commonly done).

**Reply by jlbamber** [reply]
Hi Torge, for the GrIS we can do a lot better than assume balance prior to 1970. See e.g. Box and Colgan 2013, which has been updated by Ken to produce a total mass balance (TMB) time series for 1840-next week. I mention this in my talk. It comes with uncertainties. Of course, it's not perfect but much better than assuming balance esp for the 1930-40s when it definitely wasn't. 

For the AIS it's a bit more tricky but, as I suggested in my talk, Frederiske (2020) manages to achieve SLB closure for 1900-20XX with an assumption for AIS mass trends that are pretty close to balance and I think this is pretty compelling for earlier 20th C. Jonathan Gregory did the same in an earlier study but Thomas's is a more robust reconstruction.

**Reply by mankoff** [reply]
@AndyShepCPOM 

> We have several input output (IO) datasets submitted to IMBIE and so it can be computed from those

To be pedantic, I/O as our community uses the term isn't sufficient.  You need the I to be provided with constituent terms (snowfall, rain, runoff).  If the IO provided to you is just net SMB & discharge, you've lost necessary information.

**Reply by AndyShepCPOM** [reply]
IMBIE has a surface mass balance experiment group who assess regional climate models and their various elements, including net surface mass balance

**Reply by mankoff** [reply]
Discussions on *historical balance* are ongoing on a different thread: https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/13

**Comment by durack1** [comment]
Looks like there's already a heap to collate here! For the CMIP7 DECK/CMIP historical experiment, we expect to collate forcings that cover the 1850-2025 (or the latest available) period. Providing time histories that extend back that far would be ideal if it were possible. Uncertainties will be huge (or signals will be tiny)... This is also true of other forcing datasets.

For a little more context, we have a CMIP Forcing Task Team (TT) homepage at https://wcrp-cmip.org/cmip7-task-teams/forcings/, and the current timelines that we are hoping to hit are located at https://zenodo.org/records/10326376 (and have been updated 7 times already). We have started to see the first batch of new prototype forcings land for review already. In brief, we expect to collate the first complete set of ***CMIP:DECK:historical*** forcings by April 2023 (covering the 1850-2021 time period), and through iteration and testing, update these late 2024 and 2025 accounting for feedback, and attempting to temporally extend these as far forward to near real-time as possible. The aim then is for their use in the ***CMIP7:DECK:historical*** (plus esm-historical, *piControl, etc)

**Reply by durack1** [reply]
I added a comment in discussion #13 regarding how I have been imagining the process (of timeseries generation, to model testing, to proposing this as a ***CMIP:DECK:historical*** experiment amendment) would occur - see https://github.com/NASA-GISS/freshwater-forcing-workshop/discussions/13#discussioncomment-8457655.

I hope this is helpful, happy to hear alternative views that improve on these initial thoughts

**Comment by jlbamber** [comment]
So far not been much discussion of GIC so here are a few points that came up. As for the GrIS global mass balance estimates existing for the GRACE period and reconstructions, combining glaciological, geodetic and statistical methods extend back to 1960s. Reconstructions based on re-analysis exist for earlier part of the 20th C (e.g. Marzeion et al 2012) with larger uncertainties. What proportion of the GIC imbalance enters the ocean as a FW anomaly is not well constrained for some of the RGI sectors, the most important of which is the HMA. However, these flux anoamlies (for individual RGI regions) are small compared to the ice sheets and other FW flux sources and sinks. The exception to thiss, and where the FW anomaly may be important ofr ocean processes, is in the Arctic. Over half the GIC volume lies in the Arctic and here the assumption that it enters the ocean as an anomaly is reasonable to first order. A reconstruction for 1950-2021 is available for Arctic GIC. Prior to this will require using statistical approaches such as Marzeion 2012.
